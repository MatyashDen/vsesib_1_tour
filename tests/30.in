20 20 4 6
0 0
1 1
3 19
10 0
12 13
14 8
