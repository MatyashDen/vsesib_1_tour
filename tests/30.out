3598
0RLE0DUE0RRLLE0DDUUE0RRRLLLE0DDDUUUE0RRRRLLLLE0DDDDUUUUE0RRRRRLLLLLE0DDDDDUUUUUE0RRRRRRLLLLLLE0RRRRRRRLLLLLLLE0RRRRRRRRLLLLLLLLE0RRRRRRRRRLLLLLLLLLE0RRRRRRRRRRLLLLLLLLLLE0RRRRRRRRRRRLLLLLLLLLLLEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
1RLE1DUE1RRLLE1RDULE1DDUUE1RRRLLLE1RRDULLE1RDDUULE1DDDUUUE1RRRRLLLLE1RRRDULLLE1RRDDUULLE1RDDDUUULE1DDDDUUUUE1RRRRRLLLLLE1RRRRDULLLLE1RRRDDUULLLE1RRDDDUUULLE1RDDDDUUUULE1DDDDDUUUUUE1RRRRRRLLLLLLE1RRRRRDULLLLLE1RRRRDDUULLLLE1RRRDDDUUULLLE1RRDDDDUUUULLE1RDDDDDUUUUULE1RRRRRRRLLLLLLLE1RRRRRRDULLLLLLE1RRRRRDDUULLLLLE1RRRRDDDUUULLLLE1RRRDDDDUUUULLLE1RRDDDDDUUUUULLE1RRRRRRRRLLLLLLLLE1RRRRRRRDULLLLLLLE1RRRRRRDDUULLLLLLE1RRRRRDDDUUULLLLLE1RRRRDDDDUUUULLLLE1RRRDDDDDUUUUULLLE1RRRRRRRRRLLLLLLLLLE1RRRRRRRRDULLLLLLLLE1RRRRRRRDDUULLLLLLLE1RRRRRRDDDUUULLLLLLE1RRRRRDDDDUUUULLLLLE1RRRRDDDDDUUUUULLLLE1RRRRRRRRRRLLLLLLLLLLE1RRRRRRRRRDULLLLLLLLLE1RRRRRRRRDDUULLLLLLLLE1RRRRRRRDDDUUULLLLLLLE1RRRRRRDDDDUUUULLLLLLE1RRRRRDDDDDUUUUULLLLLE1RRRRRRRRDDDUUULLLLLLLLE1RRRRRRRDDDDUUUULLLLLLLE1RRRRRRDDDDDUUUUULLLLLLE1RRRRRRRRDDDDUUUULLLLLLLLE1RRRRRRRDDDDDUUUUULLLLLLLE1RRRRRRRRDDDDDUUUUULLLLLLLLEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
2LRE2DUE2UDE2LLRRE2LDURE2LUDRE2DDUUE2UUDDE2LLLRRRE2LLDURRE2LLUDRRE2LDDUURE2LUUDDRE2DDDUUUE2UUUDDDE2LLLLRRRRE2LLLDURRRE2LLLUDRRRE2LLDDUURRE2LLUUDDRRE2LDDDUUURE2LUUUDDDRE2DDDDUUUUE2LLLLLRRRRRE2LLLLDURRRRE2LLLLUDRRRRE2LLLDDUURRRE2LLLUUDDRRRE2LLDDDUUURRE2LLUUUDDDRRE2LDDDDUUUURE2DDDDDUUUUUE2LLLLLLRRRRRRE2LLLLLDURRRRRE2LLLLLUDRRRRRE2LLLLDDUURRRRE2LLLLUUDDRRRRE2LLLDDDUUURRRE2LLLUUUDDDRRRE2LLDDDDUUUURRE2LDDDDDUUUUURE2DDDDDDUUUUUUE2LLLLLLLRRRRRRRE2LLLLLLDURRRRRRE2LLLLLLUDRRRRRRE2LLLLLDDUURRRRRE2LLLLLUUDDRRRRRE2LLLLDDDUUURRRRE2LLLLUUUDDDRRRRE2LLLDDDDUUUURRRE2LLDDDDDUUUUURRE2LDDDDDDUUUUUURE2DDDDDDDUUUUUUUE2LLLLLLLLRRRRRRRRE2LLLLLLLDURRRRRRRE2LLLLLLLUDRRRRRRRE2LLLLLLDDUURRRRRRE2LLLLLLUUDDRRRRRRE2LLLLLDDDUUURRRRRE2LLLLLUUUDDDRRRRRE2LLLLDDDDUUUURRRRE2LLLDDDDDUUUUURRRE2LLDDDDDDUUUUUURRE2LDDDDDDDUUUUUUURE2DDDDDDDDUUUUUUUUE2LLLLLLLLLRRRRRRRRRE2LLLLLLLLDURRRRRRRRE2LLLLLLLLUDRRRRRRRRE2LLLLLLLDDUURRRRRRRE2LLLLLLLUUDDRRRRRRRE2LLLLLLDDDUUURRRRRRE2LLLLLLUUUDDDRRRRRRE2LLLLLDDDDUUUURRRRRE2LLLLDDDDDUUUUURRRRE2LLLDDDDDDUUUUUURRRE2LLDDDDDDDUUUUUUURRE2LDDDDDDDDUUUUUUUURE2DDDDDDDDDUUUUUUUUUE2LLLLLLLLLDURRRRRRRRRE2LLLLLLLLDDUURRRRRRRRE2LLLLLLLDDDUUURRRRRRRE2LLLLLLLUUUDDDRRRRRRRE2LLLLLLDDDDUUUURRRRRRE2LLLLLDDDDDUUUUURRRRRE2LLLLDDDDDDUUUUUURRRRE2LLLDDDDDDDUUUUUUURRRE2LLDDDDDDDDUUUUUUUURRE2LDDDDDDDDDUUUUUUUUURE2DDDDDDDDDDUUUUUUUUUUE2LLLLLLLLLDDUURRRRRRRRRE2LLLLLLLLDDDUUURRRRRRRRE2LLLLLLLDDDDUUUURRRRRRRE2LLLLLLDDDDDUUUUURRRRRRE2LLLLLDDDDDDUUUUUURRRRRE2LLLLDDDDDDDUUUUUUURRRRE2LLLDDDDDDDDUUUUUUUURRRE2LLDDDDDDDDDUUUUUUUUURRE2LDDDDDDDDDDUUUUUUUUUURE2DDDDDDDDDDDUUUUUUUUUUUE2LLLLLLLLLDDDUUURRRRRRRRRE2LLLLLLLLDDDDUUUURRRRRRRRE2LLLLLLLDDDDDUUUUURRRRRRRE2LLLLLLDDDDDDUUUUUURRRRRRE2LLLLLDDDDDDDUUUUUUURRRRRE2LLLLDDDDDDDDUUUUUUUURRRRE2LLLDDDDDDDDDUUUUUUUUURRRE2LLDDDDDDDDDDUUUUUUUUUURRE2LDDDDDDDDDDDUUUUUUUUUUURE2DDDDDDDDDDDDUUUUUUUUUUUUE2LLLLLLLLLDDDDUUUURRRRRRRRRE2LLLLLLLLDDDDDUUUUURRRRRRRRE2LLLLLLLDDDDDDUUUUUURRRRRRRE2LLLLLLDDDDDDDUUUUUUURRRRRRE2LLLLLDDDDDDDDUUUUUUUURRRRRE2LLLLDDDDDDDDDUUUUUUUUURRRRE2LLLDDDDDDDDDDUUUUUUUUUURRRE2LLDDDDDDDDDDDUUUUUUUUUUURRE2LDDDDDDDDDDDDUUUUUUUUUUUURE2DDDDDDDDDDDDDUUUUUUUUUUUUUE2LLLLLLDDDDDDDDUUUUUUUURRRRRRE2LLLLLDDDDDDDDDUUUUUUUUURRRRRE2LLLLDDDDDDDDDDUUUUUUUUUURRRRE2LLLDDDDDDDDDDDUUUUUUUUUUURRRE2LLDDDDDDDDDDDDUUUUUUUUUUUURRE2LDDDDDDDDDDDDDUUUUUUUUUUUUURE2DDDDDDDDDDDDDDUUUUUUUUUUUUUUE2LLLLLLDDDDDDDDDUUUUUUUUURRRRRRE2LLLLLDDDDDDDDDDUUUUUUUUUURRRRRE2LLLLDDDDDDDDDDDUUUUUUUUUUURRRRE2LLLDDDDDDDDDDDDUUUUUUUUUUUURRRE2LLDDDDDDDDDDDDDUUUUUUUUUUUUURRE2LDDDDDDDDDDDDDDUUUUUUUUUUUUUURE2DDDDDDDDDDDDDDDUUUUUUUUUUUUUUUE2LLLLLLDDDDDDDDDDUUUUUUUUUURRRRRRE2LLLLLDDDDDDDDDDDUUUUUUUUUUURRRRRE2LLLLDDDDDDDDDDDDUUUUUUUUUUUURRRRE2LLLDDDDDDDDDDDDDUUUUUUUUUUUUURRRE2LLDDDDDDDDDDDDDDUUUUUUUUUUUUUURRE2LDDDDDDDDDDDDDDDUUUUUUUUUUUUUUURE2DDDDDDDDDDDDDDDDUUUUUUUUUUUUUUUUE2LLLLLLDDDDDDDDDDDUUUUUUUUUUURRRRRRE2LLLLLDDDDDDDDDDDDUUUUUUUUUUUURRRRRE2LLLLDDDDDDDDDDDDDUUUUUUUUUUUUURRRRE2LLLDDDDDDDDDDDDDDUUUUUUUUUUUUUURRRE2LLDDDDDDDDDDDDDDDUUUUUUUUUUUUUUURRE2LDDDDDDDDDDDDDDDDUUUUUUUUUUUUUUUURE2LLLLLLDDDDDDDDDDDDUUUUUUUUUUUURRRRRRE2LLLLLDDDDDDDDDDDDDUUUUUUUUUUUUURRRRRE2LLLLDDDDDDDDDDDDDDUUUUUUUUUUUUUURRRRE2LLLDDDDDDDDDDDDDDDUUUUUUUUUUUUUUURRRE2LLDDDDDDDDDDDDDDDDUUUUUUUUUUUUUUUURRE2LLLLLLDDDDDDDDDDDDDUUUUUUUUUUUUURRRRRRE2LLLLLDDDDDDDDDDDDDDUUUUUUUUUUUUUURRRRRE2LLLLDDDDDDDDDDDDDDDUUUUUUUUUUUUUUURRRRE2LLLDDDDDDDDDDDDDDDDUUUUUUUUUUUUUUUURRRE2LLLLLLDDDDDDDDDDDDDDUUUUUUUUUUUUUURRRRRRE2LLLLLDDDDDDDDDDDDDDDUUUUUUUUUUUUUUURRRRRE2LLLLDDDDDDDDDDDDDDDDUUUUUUUUUUUUUUUURRRRE2LLLLLLDDDDDDDDDDDDDDDUUUUUUUUUUUUUUURRRRRRE2LLLLLDDDDDDDDDDDDDDDDUUUUUUUUUUUUUUUURRRRRE2LLLLLLDDDDDDDDDDDDDDDDUUUUUUUUUUUUUUUURRRRRRE
3RLE3DUE3UDE3RRLLE3RDULE3RUDLE3DDUUE3UUDDE3RRRLLLE3RRDULLE3RRUDLLE3RDDUULE3RUUDDLE3DDDUUUE3UUUDDDE3RRRRLLLLE3RRRDULLLE3RRRUDLLLE3RRDDUULLE3RRUUDDLLE3RDDDUUULE3RUUUDDDLE3DDDDUUUUE3UUUUDDDDE3RRRRRLLLLLE3RRRRDULLLLE3RRRRUDLLLLE3RRRDDUULLLE3RRRUUDDLLLE3RRDDDUUULLE3RRUUUDDDLLE3RDDDDUUUULE3DDDDDUUUUUE3RRRRRRLLLLLLE3RRRRRDULLLLLE3RRRRRUDLLLLLE3RRRRDDUULLLLE3RRRRUUDDLLLLE3RRRDDDUUULLLE3RRRUUUDDDLLLE3RRDDDDUUUULLE3RDDDDDUUUUULE3DDDDDDUUUUUUE3RRRRRRRLLLLLLLE3RRRRRRDULLLLLLE3RRRRRRUDLLLLLLE3RRRRRDDUULLLLLE3RRRRRUUDDLLLLLE3RRRRDDDUUULLLLE3RRRRUUUDDDLLLLE3RRRDDDDUUUULLLE3RRDDDDDUUUUULLE3RDDDDDDUUUUUULE3DDDDDDDUUUUUUUE3RRRRRRRRLLLLLLLLE3RRRRRRRDULLLLLLLE3RRRRRRRUDLLLLLLLE3RRRRRRDDUULLLLLLE3RRRRRRUUDDLLLLLLE3RRRRRDDDUUULLLLLE3RRRRRUUUDDDLLLLLE3RRRRDDDDUUUULLLLE3RRRDDDDDUUUUULLLE3RRDDDDDDUUUUUULLE3RDDDDDDDUUUUUUULE3DDDDDDDDUUUUUUUUE3RRRRRRRRRLLLLLLLLLE3RRRRRRRRDULLLLLLLLE3RRRRRRRRUDLLLLLLLLE3RRRRRRRDDUULLLLLLLE3RRRRRRRUUDDLLLLLLLE3RRRRRRDDDUUULLLLLLE3RRRRRRUUUDDDLLLLLLE3RRRRRDDDDUUUULLLLLE3RRRRDDDDDUUUUULLLLE3RRRDDDDDDUUUUUULLLE3RRDDDDDDDUUUUUUULLE3RDDDDDDDDUUUUUUUULE3DDDDDDDDDUUUUUUUUUE3RRRRRRRRRRLLLLLLLLLLE3RRRRRRRRRDULLLLLLLLLE3RRRRRRRRRUDLLLLLLLLLE3RRRRRRRRDDUULLLLLLLLE3RRRRRRRRUUDDLLLLLLLLE3RRRRRRRDDDUUULLLLLLLE3RRRRRRRUUUDDDLLLLLLLE3RRRRRRDDDDUUUULLLLLLE3RRRRRDDDDDUUUUULLLLLE3RRRRDDDDDDUUUUUULLLLE3RRRDDDDDDDUUUUUUULLLE3RRDDDDDDDDUUUUUUUULLE3RDDDDDDDDDUUUUUUUUULE3RRRRRRRRRRRLLLLLLLLLLLE3RRRRRRRRRRDULLLLLLLLLLE3RRRRRRRRRRUDLLLLLLLLLLE3RRRRRRRRRDDUULLLLLLLLLE3RRRRRRRRRUUDDLLLLLLLLLE3RRRRRRRRDDDUUULLLLLLLLE3RRRRRRRRUUUDDDLLLLLLLLE3RRRRRRRDDDDUUUULLLLLLLE3RRRRRRDDDDDUUUUULLLLLLE3RRRRRDDDDDDUUUUUULLLLLE3RRRRDDDDDDDUUUUUUULLLLE3RRRDDDDDDDDUUUUUUUULLLE3RRDDDDDDDDDUUUUUUUUULLE3RRRRRRRRRRRRLLLLLLLLLLLLE3RRRRRRRRRRRDULLLLLLLLLLLE3RRRRRRRRRRRUDLLLLLLLLLLLE3RRRRRRRRRRDDUULLLLLLLLLLE3RRRRRRRRRRUUDDLLLLLLLLLLE3RRRRRRRRRDDDUUULLLLLLLLLE3RRRRRRRRRUUUDDDLLLLLLLLLE3RRRRRRRRDDDDUUUULLLLLLLLE3RRRRRRRDDDDDUUUUULLLLLLLE3RRRRRRDDDDDDUUUUUULLLLLLE3RRRRRDDDDDDDUUUUUUULLLLLE3RRRRDDDDDDDDUUUUUUUULLLLE3RRRDDDDDDDDDUUUUUUUUULLLE3RRRRRRRRRRRRDULLLLLLLLLLLLE3RRRRRRRRRRRDDUULLLLLLLLLLLE3RRRRRRRRRRDDDUUULLLLLLLLLLE3RRRRRRRRRDDDDUUUULLLLLLLLLE3RRRRRRRRDDDDDUUUUULLLLLLLLE3RRRRRRRDDDDDDUUUUUULLLLLLLE3RRRRRRDDDDDDDUUUUUUULLLLLLE3RRRRRDDDDDDDDUUUUUUUULLLLLE3RRRRDDDDDDDDDUUUUUUUUULLLLE3RRRRRRRRRRRRDDUULLLLLLLLLLLLE3RRRRRRRRRRRDDDUUULLLLLLLLLLLE3RRRRRRRRRRDDDDUUUULLLLLLLLLLE3RRRRRRRRRDDDDDUUUUULLLLLLLLLE3RRRRRRRRDDDDDDUUUUUULLLLLLLLE3RRRRRRRDDDDDDDUUUUUUULLLLLLLE3RRRRRRDDDDDDDDUUUUUUUULLLLLLE3RRRRRDDDDDDDDDUUUUUUUUULLLLLE3RRRRRRRRRRRRDDDUUULLLLLLLLLLLLE3RRRRRRRRRRRDDDDUUUULLLLLLLLLLLE3RRRRRRRRRRDDDDDUUUUULLLLLLLLLLE3RRRRRRRRRDDDDDDUUUUUULLLLLLLLLE3RRRRRRRRDDDDDDDUUUUUUULLLLLLLLE3RRRRRRRDDDDDDDDUUUUUUUULLLLLLLE3RRRRRRDDDDDDDDDUUUUUUUUULLLLLLE3RRRRRRRRRRRRDDDDUUUULLLLLLLLLLLLE3RRRRRRRRRRRDDDDDUUUUULLLLLLLLLLLE3RRRRRRRRRRDDDDDDUUUUUULLLLLLLLLLE3RRRRRRRRRDDDDDDDUUUUUUULLLLLLLLLE3RRRRRRRRDDDDDDDDUUUUUUUULLLLLLLLE3RRRRRRRDDDDDDDDDUUUUUUUUULLLLLLLE3RRRRRRRRRRRRDDDDDUUUUULLLLLLLLLLLLE3RRRRRRRRRRRDDDDDDUUUUUULLLLLLLLLLLE3RRRRRRRRRRDDDDDDDUUUUUUULLLLLLLLLLE3RRRRRRRRRDDDDDDDDUUUUUUUULLLLLLLLLE3RRRRRRRRDDDDDDDDDUUUUUUUUULLLLLLLLE3RRRRRRRRRRRRDDDDDDUUUUUULLLLLLLLLLLLE3RRRRRRRRRRRDDDDDDDUUUUUUULLLLLLLLLLLE3RRRRRRRRRRDDDDDDDDUUUUUUUULLLLLLLLLLE3RRRRRRRRRDDDDDDDDDUUUUUUUUULLLLLLLLLE3RRRRRRRRRRRRDDDDDDDUUUUUUULLLLLLLLLLLLE3RRRRRRRRRRRDDDDDDDDUUUUUUUULLLLLLLLLLLE3RRRRRRRRRRDDDDDDDDDUUUUUUUUULLLLLLLLLLE3RRRRRRRRRRRRDDDDDDDDUUUUUUUULLLLLLLLLLLLE3RRRRRRRRRRRDDDDDDDDDUUUUUUUUULLLLLLLLLLLE3RRRRRRRRRRRRDDDDDDDDDUUUUUUUUULLLLLLLLLLLLEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
